ITR/data/base_providers.py

```diff
@@ -178,13 +181,14 @@ def _get_intensity_benchmarks(self, company_sector_region_info: pd.DataFrame,
         :param scope: a scope
         :return: A DataFrame with company and intensity benchmarks per calendar year per row
         """
+        sectors = company_sector_region_info[self.column_config.SECTOR]
+        regions = company_sector_region_info[self.column_config.REGION].copy()
         benchmark_projection = self._get_projected_intensities(scope)  # TODO optimize performance
-        reg_sec = company_sector_region_info[[self.column_config.REGION,self.column_config.SECTOR]].copy()
-        merged_df=reg_sec.reset_index().merge(benchmark_projection.reset_index()[[self.column_config.REGION,self.column_config.SECTOR]], how='left', indicator=True).set_index('index') # checking which combinations of reg-sec are missing in the benchmark
-        reg_sec.loc[merged_df._merge == 'left_only', self.column_config.REGION] = "Global" # change region in missing combination to "Global"
-        sectors = reg_sec.sector
-        regions = reg_sec.region
-        benchmark_projection = benchmark_projection.loc[list(zip(regions, sectors))]
+        mask = regions.isin(benchmark_projection.reset_index()[self.column_config.REGION])
+        regions.loc[~mask] = "Global"
+
+        # benchmark_projection has a scope by construction
+        benchmark_projection = benchmark_projection.loc[list(zip(regions, sectors, [scope] * len(sectors)))]
```

Comment:
Same here, [scope] * len(sectors) may work in short term, but could become problematic long term, when we start supporting multiple scopes per sector